### DISCUSSION TITLE
Exclude videos based on length or filesize

### DISCUSSION THREAD

**Discussion by paswai** [immich-app/immich#12427] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Feature : Extent available exclusion pattern to include video length and/or video file size.

Example use : Live photos include short (few seconds) videos which are shown as seperate items. Removing these items from view would provide a "cleaner" library.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Dupe of #9139

If your live photos are being correctly linked then the videos shouldn't be showing up separately.

**Comment by paswai** [comment]
@bo0tzz I don't see it as duplicate. The described situation was only an example. Further situations might apply in which one wants to exclude videos (or pictures) on features aside from the name and the folder they are in. 
Here, I mentioned specifically the file size and the length.

**Reply by bo0tzz** [reply]
How is it not a duplicate?

**Reply by paswai** [reply]
Maybe my understanding of it is incorrect, but my proposed feature is applied to external libraries during the data ingestion and files exclusion via patterns. #9139 is about already ingested files. But I noticed that I should have included more details in my description, e.g. the link to the external libraries and the exlusion patterns during scan.

**Reply by bo0tzz** [reply]
External libraries weren't mentioned anywhere in the original request, so that wasn't clear ;)

Immich doesn't parse file metadata and such until after things have already been ingested, so a feature like this wouldn't really fit in how things work.
